Makefile.packaging

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/make -f
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# Variable Definitions                                                                                            #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+PACKAGES_DIR        := ./build/packages
+GITHUB_PACKAGES_DIR := ./build/github/packages
+AZURE_PACKAGES_DIR  := ./build/azure/packages
+BINARY_PATH         := $(BUILD_DIR)/$(BINARY_NAME)
+GPG_PUBLIC_KEY      := .key
+PACKAGE_BUILD       ?= 1
+PACKAGE_VERSION     := $(shell echo ${VERSION} | tr -d 'v')-$(PACKAGE_BUILD)
+TARBALL_NAME        := $(PACKAGE_PREFIX)v3.tar.gz
+
+DEB_DISTROS         ?= ubuntu-noble-24.04 ubuntu-jammy-22.04 ubuntu-focal-20.04 debian-bookworm-12 debian-bullseye-11
```

Comment:
Don't think we support 24.04 at present.

Reply:
No problem, I'll remove for now